src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderImplTest {
+    private static final String CSV_INPUT
+            = "src/test/java/core/basesyntax/resources/reportToRead.csv";
+    private static final String WRONG_FILE_PATH
+            = "src/test/java/core/basesyntax/resources/invalidHeader.cs";
+    private static FileReader fileReader;
+
+    @BeforeAll
+    static void beforeAll() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    void read_FileNameFormat_NotOk() {
+        assertThrows(RuntimeException.class,
+                () -> fileReader.read(WRONG_FILE_PATH),
+                "Invalid file name format");
+    }
+
+    @Test
+    void read_InputIsNull_Ok() {
+        assertNotNull(fileReader.read(CSV_INPUT));
+    }
+
+    @Test
+    void read_InputIsEmptyList_Ok() {
+        List<String> emptyList = new ArrayList<>();
+        assertNotEquals(emptyList, fileReader.read(CSV_INPUT));
+    }

```

Comment:
To test an empty file, create an empty file, don't use NotEquels in that case, your case tests anything, not just reading an empty file